neurom/view/view.py

```diff
@@ -722,8 +722,8 @@ def dendrogram(obj, show_diameters=True, new_fig=True, new_axes=True, subplot=Fa
     kwargs['xlim'] = [- dnd.dims[0][0] * 0.5, dnd.dims[-1][0] * 0.5 + displacement]
 
     kwargs['title'] = kwargs.get('title', 'Morphology Dendrogram')
-    kwargs['xlabel'] = kwargs.get('xlabel', '')
-    kwargs['ylabel'] = kwargs.get('ylabel', '')
+    kwargs['xlabel'] = kwargs.get('xlabel', 'micrometers (um)')
+    kwargs['ylabel'] = kwargs.get('ylabel', 'micrometers (um)')
```

Comment:
I think a "unit of length" will be a bit confusing. We are assuming the units are um, unless the user selects a different unit. I think in this case, this should be an option. @wvangeit Could you please confirm what makes more sense for you concerning your issue https://github.com/BlueBrain/NeuroM/issues/209?